iron-overlay-behavior.html

```diff
@@ -391,8 +396,9 @@
     _finishPositioning: function() {
       this.style.display = 'none';
       this.style.transform = this.style.webkitTransform = '';
-      // force layout to avoid application of transform
-      /** @suppress {suspiciousCode} */ this.offsetWidth;
+      // Force layout layout to avoid application of transform.
+      // Set offsetWidth to itself so that compilers won't remove it.
+      this.offsetWidth = this.offsetWidth;
```

Comment:
I don't know `a = a` is a exactly the kind a compiler might optimize away (much like how `a;` was optimized away). My understanding was  the compiler doesn't know what `this.offsetWidth` does, and that's why `@suppress {suspiciousCode}` was needed. Maybe ping @rictic, our human compiler, for a double check?